src/components/aiAgent/ChatHistory.tsx

```diff
@@ -0,0 +1,128 @@
+import { gql } from '@apollo/client'
+import { DateTime } from 'luxon'
+
+import { Skeleton, Typography } from '~/components/designSystem'
+import { useGetAiConversationLazyQuery, useListAiConversationsQuery } from '~/generated/graphql'
+import { ChatRole, ChatStatus } from '~/hooks/aiAgent/aiAgentReducer'
+import { useAiAgent } from '~/hooks/aiAgent/useAiAgent'
+import { tw } from '~/styles/utils'
+
+gql`
+  query getAiConversation($id: ID!) {
+    aiConversation(id: $id) {
+      id
+      name
+      messages {
+        content
+        type
+      }
+    }
+  }
+
+  query listAiConversations($limit: Int) {
+    aiConversations(limit: $limit) {
+      collection {
+        id
+        name
+        updatedAt
+      }
+    }
+  }
+`
```

Comment:
Following https://www.notion.so/getlago/GraphQL-Query-Colocation-Anti-Pattern-2a8ef63110d280859247d2e45a4dead3?source=copy_link

Please move this to a hook and use it there